lib/read_table_A.pl

```diff
@@ -1103,19 +1081,6 @@ sub read_table_A {
             $code .= "use Philips_Hue;\n";
         }
     }
-    elsif ($type eq "PHILIPS_LUX"){
```

Comment:
Hi Lieven,

I just made a change to the read_table_A.pl file and committed it (hopefully correcting my earlier mistake of removing data).  Do I need to do a new pull request for these changes to show up?  I did it in the same branch as before.

From: Lieven Hollevoet [mailto:notifications@github.com]
Sent: Sunday, November 8, 2015 3:12 PM
To: hollie/misterhouse <misterhouse@noreply.github.com>
Cc: Timothy Spaulding <spaulding@icanbrew.com>
Subject: Re: [misterhouse] New Dsc module (#550)


In lib/read_table_A.pl<https://github.com/hollie/misterhouse/pull/550#discussion_r44229725>:

> @@ -1103,19 +1081,6 @@ sub read_table_A {

>              $code .= "use Philips_Hue;\n";

>          }

>      }

> -    elsif ($type eq "PHILIPS_LUX"){
Hey Timothy, I don’t know how you exactly proceeded to create the branch, but if you start from hollie/master then you should have the correct file. Maybe you accidentally copied over a file from an older branch? Best regards, Lieven.
…
Op 26-okt.-2015, om 01:17 heeft Timothy Spaulding <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven: In lib/read_table_A.pl: > @@ -1103,19 +1081,6 @@ sub read_table_A { > $code .= "use Philips_Hue;\n"; > } > } > - elsif ($type eq "PHILIPS_LUX"){ How do I accomplish this task of pulling down the latest? To create my branches, I am sure I started from scratch with a new fork and new branches; so I don’t understand how I don’t have the latest. From: Lieven Hollevoet [mailto:notifications@github.com] Sent: Sunday, October 25, 2015 4:45 PM To: hollie/misterhouse <misterhouse@noreply.github.com<mailto:misterhouse@noreply.github.com>> Cc: Timothy Spaulding <spaulding@icanbrew.com<mailto:spaulding@icanbrew.com>> Subject: Re: [misterhouse] New Dsc module (#550<https://github.com/hollie/misterhouse/pull/550>) In lib/read_table_A.pl<#550<https://github.com/hollie/misterhouse/pull/550> (comment)>: @@ -1103,19 +1081,6 @@ sub read_table_A { $code .= "use Philips_Hue;\n"; } } - elsif ($type eq "PHILIPS_LUX"){ Hey Timothy, could you please update the pull request so that the PHILIPS_LUX support is not removed? Thank you, Lieven. — Reply to this email directly or view it on GitHub<https://github.com/hollie/misterhouse/pull/550/files#r42950129>. — Reply to this email directly or view it on GitHub.

—
Reply to this email directly or view it on GitHub<https://github.com/hollie/misterhouse/pull/550/files#r44229725>.